mfa.tf

```diff
@@ -0,0 +1,17 @@
+data "aws_caller_identity" "current" {}
```

Comment:
seems you already have created this block, prevent duplicate data blocks and reuse created one